src/cephadm/cephadm

```diff
@@ -48,10 +48,10 @@ FuncT = TypeVar('FuncT', bound=Callable)
 DEFAULT_IMAGE = 'quay.io/ceph/ceph:v17'
 DEFAULT_IMAGE_IS_MASTER = False
 DEFAULT_IMAGE_RELEASE = 'quincy'
-DEFAULT_PROMETHEUS_IMAGE = 'quay.io/prometheus/prometheus:v2.18.1'
-DEFAULT_NODE_EXPORTER_IMAGE = 'quay.io/prometheus/node-exporter:v0.18.1'
-DEFAULT_ALERT_MANAGER_IMAGE = 'quay.io/prometheus/alertmanager:v0.20.0'
-DEFAULT_GRAFANA_IMAGE = 'quay.io/ceph/ceph-grafana:6.7.4'
+DEFAULT_PROMETHEUS_IMAGE = 'quay.io/prometheus/prometheus:v2.33.4'
+DEFAULT_NODE_EXPORTER_IMAGE = 'quay.io/prometheus/node-exporter:v1.3.1'
+DEFAULT_ALERT_MANAGER_IMAGE = 'quay.io/prometheus/alertmanager:v0.23.0'
+DEFAULT_GRAFANA_IMAGE = 'quay.io/ceph/ceph-grafana:8.3.5'
```

Comment:
@epuertat changes look good to me,
Are we planning to backport this upgraded grafana and monitoring stack versions to pacific as well?

Reply:
@sunilangadi2 potentially any user could have this in Pacific just by updating their DEFAULT_GRAFANA_IMAGE to this version (as it also includes the embedded Grafana Dashboards).

In any case @aaSharma14 is working to remove the embedded Grafana Dashboards from the container, and use a vanilla Grafana container (no more custom upstream/downstream builds of ceph-grafana container). Please @aaSharma14 add the trackers for these work items.

Reply:
https://tracker.ceph.com/issues/53275